cyber-security/modules/csls_s3_object_logging/vars.tf

```diff
@@ -0,0 +1,31 @@
+variable "cloudwatch_destination_arn" {
+  type        = string
+  description = "Ask for the CSLS cloudwatch destination ARN in #cyber-security-help."
+}
+
+variable "cloudwatch_filter_pattern" {
+  type        = string
+  description = "Can be used to filter events if required."
+  default     = ""
+}
+
+variable "bucket_arn_list" {
+  type        = list(string)
+  description = "By default log all S3 buckets in the account"
+  default = [
+    "arn:aws:s3:::"
+  ]
+}
+
+variable "logging_suffix" {
+  type        = string
+  description = "For partial logging this is appended to the trail, log bucket name and cloudwatch log group."
+  default     = ""
+}
+
+variable "tags" {
```

Comment:
Maybe add some module level tags to link back to this module.

Reply:
Done

Reply:
I did this as a merge of var.tags into local.tags and then applied local.tags so that we always get the default module tags regardless of what other tags the user applies.